### DISCUSSION TITLE
V23.3 "edit configuration" "tooltips" missing option names

### DISCUSSION THREAD

**Discussion by ll15039** [opendcim/openDCIM#1446] [Q&A] [answered]
Upgraded from 4.3 to 18 then 18 to 23.3. What I've seen so far, most everything works except:

Noticed in "Edit Configuration" -> "ToolTips"  none of the ToolTips names are offered for selection, completely blank.
ToolTips do work when mousing over the device, likely because they were already set in version 4.3.

Thank-you for your help.

![Screenshot 2023-09-23 at 00-53-45 openDCIM Data Center Inventory](https://github.com/opendcim/openDCIM/assets/19232018/e1c178da-f870-46f6-99d0-eeaeb2b16793)

**Comment by wilpig** [comment]
Output of?
```
SELECT * FROM fac_CabinetToolTip ORDER BY SortOrder ASC, Enabled DESC, Label ASC;
```

**Reply by ll15039** [reply]
<pre>
mysql> use dcim;
Reading table information for completion of table and column names
You can turn off this feature to get a quicker startup with -A

Database changed
mysql> SELECT * FROM fac_CabinetToolTip ORDER BY SortOrder ASC, Enabled DESC, Label ASC;
+-----------+------------------+-----------------------------+---------+
| SortOrder | Field            | Label                       | Enabled |
+-----------+------------------+-----------------------------+---------+
|      NULL | AssetTag         | Asset Tag                   |       0 |
|      NULL | EscalationID     | Details                     |       0 |
|      NULL | DeviceID         | Device ID                   |       0 |
|      NULL | Status           | Device Status               |       0 |
|      NULL | InstallDate      | Install Date                |       0 |
|      NULL | MfgDate          | Manufacture Date            |       0 |
|      NULL | NominalWatts     | Nominal Draw (Watts)        |       0 |
|      NULL | Ports            | Number of Data Ports        |       0 |
|      NULL | PowerSupplyCount | Number of Power Supplies    |       0 |
|      NULL | ChassisSlots     | Number of Slots in Chassis: |       0 |
|      NULL | PrimaryContact   | Primary Contact             |       0 |
|      NULL | PrimaryIP        | Primary IP                  |       0 |
|      NULL | SNMPCommunity    | SNMP Read Only Community    |       0 |
|      NULL | EscalationTimeID | Time Period                 |       0 |
|      NULL | Hypervisor       | VM Hypervisor               |       0 |
|      NULL | WarrantyCo       | Warranty Company            |       0 |
|      NULL | WarrantyExpire   | Warranty Expiration         |       0 |
|         0 | Owner            | Departmental Owner          |       1 |
|         1 | TemplateID       | Device Class                |       1 |
|         2 | DeviceType       | Device Type                 |       1 |
|         3 | SerialNo         | Serial Number               |       1 |
+-----------+------------------+-----------------------------+---------+
21 rows in set (0.01 sec)
</pre>

**Comment by wilpig** [comment]
You have a javascript or style problem.  Check that you have the latest versions of all the files in place and didn't keep something old.

**Reply by ll15039** [reply, marked as answer]
Tried a [shift] refresh in firefox 102.15.0esr (64-bit) cleared the none display issue.
